### PR TITLE
Enhance code style with prettier and husky

### PR DESCRIPTION
**Pull request by shivam-sharma7**
- This https://github.com/profilecity/vidur/commit/f98270afd04dfea3562cb3c180db0b2e7ad4094c commit reffers to config of prettier and 
- This https://github.com/profilecity/vidur/commit/a5e414f9ef54e2a6d674b4a89074e173ae0e6472 commit reffers to latest code style depend on `.prettierrc` file.

- fixes: #94 

Also, now it would format only edited file if you run `yarn format` otherwise it won't, this is why i formatted codebase in this commit https://github.com/profilecity/vidur/commit/a5e414f9ef54e2a6d674b4a89074e173ae0e6472

### CODE REVIEW

**Comment by amandesai01** [comment]
That's amazing, do that! thanks!

**Comment by amandesai01** [comment]
Don't format files in this PR

**Comment by shivam-sharma7** [comment]
> Don't format files in this PR

Done

**Comment by amandesai01** [comment]
Also add post commit hook

**Comment by shivam-sharma7** [comment]
> Also add post commit hook

I think you want to add `pre-commit` hook as we have discussed in https://github.com/profilecity/vidur/issues/94#issuecomment-2298672349

**Comment by shivam-sharma7** [comment]
> Don't format files in this PR

Remember once this PR get merged then whenever you will change files or doing some work then when you try to commit those changes then it will format that file but also format unformatted file. Attention this is why I have formatted all because once all codebase follow our new rule then it will not change randomly, this will change only files that has changed by you or someone..

**Comment by shivam-sharma7** [comment]
@amandesai01 Now your changes in `main` branch, you can merge this

**Comment by amandesai01** [comment]
It's still undoing my changes, see package.json. The release command is getting deleted.

**Comment by shivam-sharma7** [comment]
> It's still undoing my changes, see package.json. The release command is getting deleted.

got it, check now

File: package.json [hunk 1]
```diff
@@ -44,6 +46,9 @@
   },
   "devDependencies": {
     "@types/pg": "^8.11.6",
+    "husky": "^9.1.5",
+    "prettier": "^3.3.3",
     "changelogen": "^0.5.5"
+
```
Comment (amandesai01) [inline package.json:52]:
```suggestion
```

**Review by amandesai01 — CHANGES_REQUESTED** [review]

**Comment by amandesai01** [comment]
Done, will merge in some time.

**Comment by amandesai01** [comment]
Can we have github action too in this PR?

**Comment by shivam-sharma7** [comment]
> Can we have github action too in this PR?

I have another issue and why i have this please check here -https://github.com/profilecity/vidur/issues/109#issuecomment-2322934715

**Review by amandesai01 — APPROVED** [review]

**Pull request merged**